disnake/channel.py

```diff
@@ -871,10 +871,10 @@ async def create_thread(
 
             .. versionadded:: 2.3
 
-        slowmode_delay: :class:`int`
+        slowmode_delay: Optional[:class:`int`]
             Specifies the slowmode rate limit for users in this thread, in seconds.
             A value of ``0`` disables slowmode. The maximum value possible is ``21600``.
-            If not provided, slowmode is disabled.
+            If not provided or ``None``, slowmode is inherited from the parent channel.
```

Comment:
the or none part here is confusing IMO

Reply:
https://github.com/DisnakeDev/disnake/pull/818/commits/66d6206b059d29ce1f4cf1de09ed1ee2c6814dcc